runtimes/rust/src/lib.rs

```diff
@@ -296,7 +294,7 @@ pub mod _reexports {
     //! Types necessary for `ServiceDispatcher` and `ServiceFactory` derive macros to work.
 
     pub use exonum::runtime::{
-        ArtifactId, CommonError, ExecutionError, MethodId, RuntimeIdentifier,
+        ArtifactId, CallContext, CommonError, ExecutionError, MethodId, RuntimeIdentifier,
```

Comment:
I think that is more convenient to export `CallContext` as `pub use exonum::runtime::CallContext` because `Service` documentation refers on the `CallContext` structure, therefore if I don't write this reexport I will lose this link.

Reply:
Why can't you just paste a link as `https://docs.rs/exonum/latest/exonum/runtime/struct.CallContext.html`?

Reply:
It will be more fragile in corner cases.